src/components/AppButton.tsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
-import { StyleSheet, Text, TouchableOpacity } from 'react-native';
+import {
+  GestureResponderEvent,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+} from 'react-native';
 
 type AppButtonProps = {
   title: string;
-  onPress?: any;
+  onPress: ((event: GestureResponderEvent) => void) | undefined;
```

Comment:
🔧 Infelizmente não está correto 😅. Na verdade basta definir como uma função sem parâmetros mesmo. Sei que é confuso por que o react native permite criar um botão que não tenha onPress e que receba um evento como parâmetro, mas no dia a dia só precisamos declarar parâmetros caso a função espere isso.
```suggestion
  onPress: () => void;
```

Reply:
poxa! 🤣